release-matrix/release-matrix-schema.json

```diff
@@ -0,0 +1,36 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "title": "MKE 4 Release Matrix",
+  "description": "",
+  "type": "array",
+  "items": {
+    "type": "object",
+    "properties": {
+      "clusterVersion": {
```

Comment:
nit: perhaps this should be `release` ,  `releaseVersion` or `mkeReleaseVersion`? It wasn't immediately obvious what this version represented to me.

Reply:
I went with `mkeReleaseVersion` from your suggestions. I didn't want to cause confusion between `mkectlVersion`, `mkeOperatorVersion` and `mkeVersion` originally